salt/modules/opkg.py

```diff
@@ -550,9 +550,16 @@ def install(name=None,
     if not cmds:
         return {}
 
+    updated_feeds = {}
     if refreshdb:
-        refresh_db()
-
+        updated_feeds = refresh_db()
```

Comment:
This makes me think that the function returns only the updated feeds which is not the case. Rename the variable to something less misleading. EX: update_feeds_result idk